metadata-ingestion/src/datahub/ingestion/source/looker/looker_lib_wrapper.py

```diff
@@ -206,3 +232,15 @@ def search_dashboards(
             deleted=deleted,
             transport_options=self.transport_options,
         )
+
+    def search_looks(
+        self, fields: Union[str, List[str]], deleted: Optional[bool]
+    ) -> List[Look]:
+        self.client_stats.search_looks_calls += 1
+        return list(
+            self.client.search_looks(
+                fields=self.__fields_mapper(fields),
```

Comment:
just an fyi - python convention is to use a single `_` for private methods. the double underscore `__method` has a different meaning and should be avoided